handbook/recruitment/overview.md

```diff
@@ -25,6 +25,8 @@ To provide support to our less experienced members and ensure a productive envir
 * around 40% of members be "beginners".
 * at least 20% of members (including leads) be "experienced".
 
+Each lead typically leads 1 team. We aim for a club size of 10 developers per team, to accomodate for members potentially leaving throughout the semester.
```

Comment:
Historically what was the drop rate like? And we are in uncharted territories this year since all is online so not sure if it'll be higher or lower. Anyways, if an estimate of 10 helps us to eventually reach about 5-7 then I'm happy with it.

Reply:
Sadly, I don't know if we have numbers on drop rate, since IIRC we don't take attendance (for good reason, IMO) - though I think we have some anecdotal numbers (but that varies significantly from team to team)

@renehuang1999 do you have any ideas?

Reply:
we don't have an accurate drop rate, but last year, we cap the number of people at 60-65, and there were 8 teams last year. and just to confirm, 10 is including the lead and the designer right?

Reply:
We don't factor designers, since not all teams will have designers and designers will have sandy's support as well

Reply:
8b7c3f46922c916b6659eb5b6536f7a41a137aee